procrustes/test/test_psdp.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+# The Procrustes library provides a set of functions for transforming
+# a matrix to make it as similar as possible to a target matrix.
+#
+# Copyright (C) 2017-2022 The QC-Devs Community
+#
+# This file is part of Procrustes.
+#
+# Procrustes is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# Procrustes is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, see <http://www.gnu.org/licenses/>
+#
+# --
+r"""Testings for PSDP (positive semi-definite Procrustes) module."""
+
+import numpy as np
+from numpy.testing import assert_almost_equal
+from procrustes.psdp import psdp_woodgate
+
+
+def test_psdp_identity(n=np.random.randint(50, 100)):
+    r"""Test PSDP with identity matrix."""
+    a = np.eye(n)
+    b = np.eye(n)
+    out = psdp_woodgate(a=a, b=b)
+    res = out.__getattr__("s")
+    assert_almost_equal(res, np.eye(n))
```

Comment:
I would put 

```python
res = psdp_woodgate(a=a, b=b)
s = res["s"]
error_psdp = res["error"]
assert_almost_equal(s, np.eye(n))
assert_almost_equal(error_pdsp, 0)
```

Here are the reasons. 

- There is nothing wrong to use `out` to save `ProcrustesResult` object, but we just want to make things consistent. 
- For accessing the positive-semidefinite matrix, slicing is just easier to read. 
- I also added the test of the error value, which should be done for the rest of all other tests. This would help people know how big the error is.
@banrovegrie

Reply:
Yeah, this is more readable. I will fix this in the next 15 minutes and run tox. Once tests pass, I will push and that will conclude this PR.

@FanwangM

Reply:
Btw, the code structure of the second algorithm is also complete, however some doubts persist.